Changed bootstrap  grid from 4+ 4+ 4 to 2 + 8 +2

DonaldTrump88 opened the pull request:
Default bootstrap grid is 4+ 4+ 4. It creates small middle section. To increase it the column parameters are specified.

Pull Request for Issue # .

### Summary of Changes



### Testing Instructions



### Expected result



### Actual result



### Documentation Changes Required

DonaldTrump88 committed:
Changed bootstrap  grid from 4+ 4+ 4 to 2 + 8 +2

Default bootstrap grid is 4+ 4+ 4. It creates small middle section. To increase it the column parameters are specified.

SharkyKZ commented:
Cassiopeia doesn't use Bootstrap grip.

Ninja-007 commented:
@SharkyKZ I had used fluid style in container. Then it took 4 + 4 + 4. The component area did not have enough space and sidebars had extra space. When column width is not specified then booststrap allocates equal space to all columns. BTW, the pull request code creates problem for small screen size. 


The sidebar space should be reduced to 15% +15% from 25% + 25%.

DonaldTrump88 committed:
Made index.php bootstrap grid compatible

Due to margin/padding for sidebars, it little bit tricky to adjust right right size. Ideally for sm grid should be 6-12-6 grid. In that case side-bar-left and sidebar-right should be in row which I could not get due to the padding/margin.

mbabker commented:
As was mentioned before, Cassiopeia does NOT use Bootstrap's grid system.  It is using CSS Grid.  As long as this PR is introducing Bootstrap's grid into the template, it is invalid.  If you want to change aspect's of the template's grid, you will need to update the definition in the appropriate SCSS files.

Ninja-007 commented:
@mbabker Thanks
I shall do that.
